src/services/auth.service.ts

```diff
@@ -14,12 +14,12 @@ class AuthService extends Repository<UserEntity> {
     super();
   }
 
-  public createToken(data: BaseUserResponseDTO, type: 'access' | 'refresh'): TokenData {
+  public createToken(data: BaseUserResponseDTO, type: 'access' | 'refresh' | 'shortSpanTestAccess' | 'shortSpanTestRefresh'): TokenData {
```

Comment:
nit: I think it might be good to make this an enum of the types , to we make sure that the type being passed here is valid. Then you can just do if type == types.access or whatever. Up to you

Reply:
I would stray away from using enums. I was watching a couple of videos that were saying enums are garbage in typescript and you should just use string literals instead.(This was coming from Typescript Creators) I think if you were to consolidate this type, just create a 
type TokenType = 'access' | 'refresh'